bee-node/build.rs

```diff
@@ -36,3 +89,60 @@ fn main() -> Result<(), BuildError> {
 
     Ok(())
 }
+
+#[cfg(feature = "dashboard")]
+mod dashboard {
+    use super::*;
+
+    use sha2::{Digest, Sha256};
+    use zip::ZipArchive;
+
+    use std::{
+        io::{self, BufReader, Cursor},
+        path::Path,
+    };
+
+    const RELEASE_URL: &str =
+        "https://github.com/iotaledger/node-dashboard/releases/download/v1.0.0/node-dashboard-bee-1.0.0.zip";
+    const RELEASE_CHECKSUM: &str = "d61bce7d70b51ea88536b22c5341ccdc3c4651bf8e02e093a771312f3f046c30";
+
+    pub(super) fn fetch<P: AsRef<Path>>(dashboard_dir: P) -> Result<(), BuildError> {
+        println!("downloading latest dashboard release from {}", RELEASE_URL);
+
+        let client = reqwest::blocking::Client::builder()
+            .user_agent("bee-fetch-dashboard")
+            .build()
+            .expect("could not create client");
+
+        let mut archive = tempfile::tempfile().expect("could not create temp file");
```

Comment:
same question here

Reply:
I'm not sure we really need another error variant for this, since it shouldn't really happen, and is pretty self-explanatory if it does ever get triggered, but I'm fine with adding one if we feel it's necessary.